util/Pos/Util/Filesystem.hs

```diff
@@ -107,13 +107,13 @@ withTempFile :: (MonadIO m, MonadMask m)
              -- ^ Callback that can use the file
              -> m a
 withTempFile tmpDir template action =
-  MC.bracket
+  E.bracket
     (liftIO (openTempFile tmpDir template))
     (\(name, handle) -> liftIO (hClose handle >> ignoringIOErrors (removeFile name)))
     (uncurry action)
   where
-     ignoringIOErrors :: MC.MonadCatch m => m () -> m ()
-     ignoringIOErrors ioe = ioe `MC.catch` (\e -> const (return ()) (e :: Prelude.IOError))
+     ignoringIOErrors :: E.MonadCatch m => m () -> m ()
+     ignoringIOErrors ioe = ioe `E.catch` (\e -> const (return ()) (e :: Prelude.IOError))
```

Comment:
Both `catch` and `bracket` from universum are safe, `MonadMask` and `MonadCatch` are too.